sdk/src/gateway/client.ts

```diff
@@ -191,11 +193,27 @@ export class GatewayApiClient {
             throw new Error('Invalid OP_RETURN hash');
         }
 
+        let opReturnData = data.opReturnHash;
```

Comment:
I was more imagining returning a complete psbt including the inputs through this: https://github.com/bob-collective/bob/blob/master/sdk/src/wallet/utxo.ts#L32

Just having the outputs requires the frontend then to find the inputs etc so they can't just submit this to be signed by the wallets.